etcdmain/etcd.go

```diff
@@ -160,6 +160,12 @@ func Main() {
 				}
 				os.Exit(1)
 			}
+			if strings.Contains(err.Error(), "error joining discovery cluster") {
+				plog.Errorf("%v", err)
+				plog.Infof("Please check that discovery service functions well,")
```

Comment:
Originally i thought that users should know about this through the error message. Will improve.